src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Contact.tsx

```diff
@@ -161,7 +163,7 @@ export const Contact = ({ incident, showPhone }: Props) => {
               variant="application"
             />
           )}
-          {showPhone ? (
+          {showPhone && incident.reporter.phone ? (
```

Comment:
How is this improve accessibility?

Reply:
cause you can't have an empty link on a page